magnum-zio/src/main/scala/com/augustnagro/magnum/magzio/Transactor.scala

```diff
@@ -75,6 +74,7 @@ class Transactor private (
 end Transactor
 
 object Transactor:
+  private val noOpConnectionConfig: Connection => Unit = _ => ()
```

Comment:
This avoid to allocate this function more than once when the `connectionConfig` is not set by the user